README.md

```diff
@@ -103,6 +103,32 @@ terminus(server, options);
 server.listen(PORT || 3000);
 ```
 
+## How to set Terminus up with Kubernetes?
+
+When a pod is deleted, Kubernetes will notify it and will wait for `gracePeriod` seconds before killing it.
+
+During that time window (30 seconds by default), the pod is in the `terminating` state and will be removed from any Services by a controller. The pod itself needs to catch the `SIGTERM` signal, and start failing any readiness probes.
```

Comment:
Same q as below -- Do i only need to be concerned about this with the nginx ingress controller?  If I've configured my `Service` to use a NodePort or something, then I don't need to worry, correct?